lib/internal/once.js

```diff
@@ -1,8 +1,10 @@
 export default function once(fn) {
-    return function () {
+    function wrapped () {
         if (fn === null) return;
         var callFn = fn;
         fn = null;
         callFn.apply(this, arguments);
     };
+    wrapped.promise = fn.promise;
```

Comment:
Should the wrapped function not return a promise instead to allow `wrapped().then(...)` or how do we intend this to work?